system_test/aest_sync_SUITE.erl

```diff
@@ -444,9 +454,14 @@ sync_speed(Cfg) ->
 
 node_name(I) -> list_to_atom("node" ++ integer_to_list(I)).
 
-get_block(NodeName, Height, Cfg) ->
-    Query = #{height => Height},
-    case http_get(NodeName, ext_http, [v2, 'block-by-height'], Query, Cfg) of
+get_block(NodeName, Height, _Cfg) ->
+    case request(NodeName, 'GetBlockByHeight', #{height => Height}) of
         {ok, 200, Block} -> Block;
         {ok, 404, _} -> undefined
     end.
+
+request(Node, Id, Params) ->
+    aehttp_client:request(Id, Params, 
```

Comment:
I'm not opposed to sharing the HTTP client from the integration tests, but I'm worried we're having yet another way to check for similar things (like height). Should we make some consolidation effort?

Reply:
Yes, I consolidate towards using the swagger endpoints. This would have rescued us from continuing to use a depracated endpoint in tests.

Reply:
What do we do about it? New chore in Pivotal?